gocourse05/zooobservation/camera/camera.go

```diff
@@ -0,0 +1,90 @@
+package camera
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/sarff/prjctr-golang_beginning/gocourse05/zooobservation/animal"
+)
+
+type Direction string
+
+const (
+	Left   Direction = "Left"
+	Right  Direction = "Right"
+	Top    Direction = "Top"
+	Bottom Direction = "Bottom"
+)
+
+type HistoryItem struct {
+	Time      time.Time
```

Comment:
`Time` - занадто загальна назва для поля, можна уточнити:
```suggestion
	CreatedAt time.Time
```